satellite/peer.go

```diff
@@ -351,7 +351,7 @@ func New(log *zap.Logger, full *identity.FullIdentity, db DB, config *Config, ve
 		pb.RegisterBandwidthServer(peer.Server.GRPC(), peer.Agreements.Endpoint)
 	}
 
-	{ // setup datarepair
+	if config.Repairer.Interval > 0 && config.Checker.Interval > 0 { // setup datarepair
```

Comment:
We can change `Cycle` to handle this, rather than individually specify it.

Reply:
Oh okay, let me take a look into that.